spanner/ddl/ast.go

```diff
@@ -282,6 +282,14 @@ type Foreignkey struct {
 	OnUpdate       string
 }
 
+// InterleavedParent encodes the following DDL definition:
+//
+//	INTERLEAVE IN PARENT parent_name ON DELETE delete_rule
```

Comment:
Indentation of the comment is off

Reply:
idk why but comments above earlier functions followed this indentation so I did it too, lmk if it still needs to be changed